owlbot.py

```diff
@@ -90,7 +90,7 @@
 s.remove_staging_dirs()
 
 # only run post processor when there are changes to the generated code
-if has_generator_updates:
+if True or has_generator_updates:
```

Comment:
This is needed to apply the template changes. I'll revert this change once owlbot runs.